Makefile

```diff
@@ -301,10 +301,6 @@ build-and-deploy-ui:
 prepare-for-e2e:
 	$(call workspace_bundle,base) \
 	&& $(call workspace_service_bundle,guacamole) \
-	&& $(call workspace_service_bundle,azureml) \
```

Comment:
You've removed also azureml, gitea and innereye intentionally?

Reply:
yes, they are not tested in any e2e so we don't need to build, publish and register them